tools/eslint-excludelist.json

```diff
@@ -102,7 +102,6 @@
 	"projects/plugins/jetpack/extensions/blocks/opentable/use-restaurant-search.js",
 	"projects/plugins/jetpack/extensions/blocks/pinterest/utils.js",
 	"projects/plugins/jetpack/extensions/blocks/podcast-player/utils.js",
-	"projects/plugins/jetpack/extensions/blocks/recurring-payments/index.js",
```

Comment:
Do you happen to know why this file was excluded in the first place?

Reply:
Yep - it was related to the Product typedef, I removed the typdef to that file and moved it to `projects/plugins/jetpack/extensions/blocks/recurring-payments/button/index.js`, rewriting it in the process to make the linter happy.

Reply:
The linter was complaining about having tabs (or spaces?) in the typdef, and another rule I can't remember.